main.py

```
@@ -3,6 +3,9 @@
 import threading
 from tkinter import messagebox
 from tkinter import simpledialog
+from PIL import Image, ImageTk
+import asyncio
```

Ruff

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)